blobfuse/include/BlobfuseGlobals.h

```diff
@@ -84,6 +85,7 @@ struct cmdlineOptions
     const char *help; // print blobfuse usage
     const char *concurrency; // Max Concurrency factor for blob client wrapper (default 40)
     const char *cache_size_mb; // MAX Size of cache in MBs
+    const char *block_list_on_mount_for_secs; // Block the list api call on mount for n seconds
```

Comment:
the purpose is to not do the default list so why use time here?

Reply:
we need to block the list call for X seconds post mount. We do not have any way to determine its an implicit list by fuse driver or user driven operation, so blocking all list for few seconds after mount.

Reply:
blocking alone will not be enough, after mount it should not list at all.

Reply:
Updated.